test/functional/apps/visualize/_inspector.ts

```diff
@@ -34,6 +37,45 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await PageObjects.timePicker.setDefaultAbsoluteRange();
     });
 
+    describe('advanced input JSON', () => {
+      it('should match JSON string', async () => {
+        log.debug('Add Max Metric on memory field');
+        await PageObjects.visEditor.clickBucket('Y-axis', 'metrics');
+        await PageObjects.visEditor.selectAggregation('Max', 'metrics');
+        await PageObjects.visEditor.selectField('memory', 'metrics');
+
+        log.debug('Add value to advanced JSON input');
+        await PageObjects.visEditor.toggleAdvancedParams('2');
+        await testSubjects.setValue('codeEditorContainer', '{ "missing": 10 }');
+        await PageObjects.visEditor.clickGo();
+
+        await inspector.open();
+        await inspector.openInspectorRequestsView();
+        const requestTab = await inspector.getOpenRequestDetailRequestButton();
+        await requestTab.click();
+        const requestJSON = await inspector.getCodeEditorValue();
+
+        expect(requestJSON).to.be(
+          '{\n  "aggs": {\n    "2": {\n      "max": {\n        "field": "memory",\n        "missing": 10\n      }\n' +
```

Comment:
just interesting can we do something with it?  what if we parse `JSON` string using `JSON.parse` and then check only the advanced input value?